solutions/CEOI 10-tower.mdx

```diff
@@ -0,0 +1,41 @@
+---
+id: ceoi-10-tower
+source: CEOI 2010
+title: A Huge Tower
+author: Óscar Garries
+---
+
+[Official Analysis](http://ceoi2010.ics.upjs.sk/pub/documents/ceoi_summary.pdf#page=14)
```

Comment:
Looks good but is there any chance you can add an explanation? The one they give seems kinda long and wordy so it would be nice.

Reply:
I don't think the official one is that bad. The full-points solution is only 4 paragraphs (the rest explains sub-optimal solutions)

Reply:
4 paragraphs is long for me 😆

Reply:
I think the explanation for 100 points it's fine but the solution with all the subtask is a little messy. Maybe add something like just look the final paragraph for the 100 points explanation ?

Reply:
you know what, it doesn't matter that much. It's fine without an explanation.